test/unit/utils/genrateCloudFlareHeaders.test.js

```diff
@@ -0,0 +1,16 @@
+const { expect } = require("chai");
+const { genrateCloudFlareHeaders } = require("../../../utils/discord-actions");
+
+describe("genrateCloudFlareHeaders", function () {
+  it("generates headers with prop Content-Type and Authorization", function () {
```

Comment:
I would say prefer not to use shortnames, it might be confusing to someone else
```suggestion
  it("generates headers with property Content-Type and Authorization", function () {
```